client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -561,6 +562,20 @@ export default function WPCheckout( {
 								) }
 							</>
 						}
+						completeStepContent={
```

Comment:
This should not work... something must be wrong. I think there's a bug which is causing this step to somehow be complete even though you can see from its `isCompleteCallback` below that it's not possible for it to be complete.

I made an issue and I'm going to remove this commit. https://github.com/Automattic/wp-calypso/issues/79185